domain_scoring/domain_scoring.py

```diff
@@ -17,19 +18,30 @@ def __init__(self):
         """
         # The token_pattern also allows single character strings which the default doesn't allow
         self.vectorizer = TfidfVectorizer(analyzer='word', token_pattern='\\b\\w+\\b')
-        self.classifier = DecisionTreeClassifier()
+        self.classifier = DecisionTreeClassifier(random_state=42)
         self.domain_value_transformer = NaiveTransformer()
 
-    def fit(self, metapath_graph: MetaPathRatingGraph) -> None:
+    def fit(self, metapath_graph: MetaPathRatingGraph, test_size: float = False) -> None:
         """
         Fits a classifier to predict a meta-path ordering.
+        :param test_size: Specify size of test set if a test accuracy should be reported
```

Comment:
I would put this after `metapath_graph` and mention that if left away than no accuracy is reported.